### PR TITLE
[Feat] Add Documentation In List Actions Api

### PR DESCRIPTION
**Pull request by nitishmittal1990**
## Description:
Add documentation as part of list-action API along with Commands and Params

Closes: #59 

### Implemented Following:
- [x] Use filesystem to fetch doc from mustache files
- [x] Update response to have content fetched from doc files
- [x] Display it on interactive pause on action selection.

### Screen Recording For Reference:
doc

### CODE REVIEW

File: lib/api/list-actions.js [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
 const codeceptjsFactory = require('../model/codeceptjs-factory');
 const { container } = codeceptjsFactory.getInstance();
 const { getParamsToString } = require('codeceptjs/lib/parser');
 const { methodsOfObject } = require('codeceptjs/lib/utils');
 
 module.exports = (req, res) => {
+  const docsWebApiFolderPath = path.join(__dirname, '../../node_modules/codeceptjs/docs/webapi');
+  const docFileList = [];
+  fs.readdirSync(docsWebApiFolderPath).map(fileName => {
+    docFileList.push(fileName.split('.')[0]);
```
Comment (DavertMik) [inline lib/api/list-actions.js]: I think you can use `fs.baseName` or similar function here

Comment (nitishmittal1990) [reply lib/api/list-actions.js]: yes, let me change this part of the code.

**Comment by nitishmittal1990** [comment]
> 1. move documentation below field and button
> 2. use markdown component to display it. We use markdown and we have code pieces there, they need to be formatted properly

For this, I will create a separate Issue.

File: lib/api/list-actions.js (hunk 1 above)
Comment (nitishmittal1990) [reply lib/api/list-actions.js]: Did the required change.

**Comment by DavertMik** [comment]
I merged #64 but now this one has conflict
@nitishmittal1990 please check, maybe this one needs to be closed?

**Comment by nitishmittal1990** [comment]
> I merged #64 but now this one has conflict
> @nitishmittal1990 please check, maybe this one needs to be closed?

I will resolve the conflict and update

**Comment by nitishmittal1990** [comment]
> I merged #64 but now this one has conflict
> @nitishmittal1990 please check, maybe this one needs to be closed?

Actually #63  was supposed to be merged first and then #64 . 
But I just changes a class name, so resolve the conflict issue.
Now essentially there is no difference with the master, so either we can close this PR, or simply merge it with master.

@DavertMik Please take the call. 😃

**Pull request merged**